Userland/Libraries/LibGUI/CommandPalette.cpp

```diff
@@ -253,7 +253,7 @@ void CommandPalette::collect_actions(GUI::Window& parent_window)
                 collect_actions_from_menu(*menu_item.submenu());
 
             auto const* action = menu_item.action();
-            if (action && action->is_enabled())
+            if (action && action->is_enabled() && action->text() != "Search...")
```

Comment:
This seems overly generic, doing this for _any_ "Search..." action can be unexpected

Reply:
Should I check for the menu name too or is there a better way to identify an action?

Reply:
I made it check the shortcut instead.